compose/compose-file.md

```diff
@@ -409,6 +434,15 @@ options and tags it with the specified tag.
 > **Note**: In the [version 1 file format](compose-file.md#version-1), using `build` together
 > with `image` is not allowed. Attempting to do so results in an error.
 
+### isolation
+
+> [Added in version 2.1 file format](#version-21).
```

Comment:
Are you having luck specifying just the `#tag` without the filename in it? I think in order for GitHub's markdown previewer AND our rendered site work, it is necessary for anchor links to include the file name, due to our use of relative links.  Sorry. :/ 

See: The other examples with URLs like `compose-file.md#tag`

Try staging the docs (just run `docker-compose up` in the root of your clone of this repo) and see if these links work on your localhost server. If they do, great; if not, this PR has a few tweaks in store.